docs/user/getting-started.md

```diff
@@ -97,13 +121,14 @@ $ kubectl get pods -n flightctl
 [...]
 ```
 
-### flightctl in ACM
+#### Flight Control in ACM
 
 Create a values.yaml file with the following content
 
 ```yaml
 global:
   target: acm
+  exposeServicesMethod: route
```

Comment:
```suggestion
  exposeServicesMethod: route
```

could we assume route in this case?

Reply:
right, we can!

Reply:
fixed. Now it is enough to specify `--set global.target=acm`